{"name": "Unsuccessfully login with email: simeon.hhl.qa@gmail.com and password: Test#123", "status": "passed", "steps": [{"name": "Check for error", "status": "passed", "start": 1730604435140, "stop": 1730604435375}, {"name": "Check the error text", "status": "passed", "start": 1730604435375, "stop": 1730604435382}], "parameters": [{"name": "email", "value": "'simeon.hhl.qa@gmail.com'"}, {"name": "password", "value": "'Test#123'"}, {"name": "error", "value": "'There was a problem logging you in. Please try again.'"}, {"name": "driver", "value": "'firefox'"}], "start": 1730604432287, "stop": 1730604435382, "uuid": "520e6ed3-da25-45d8-bc30-97b05169c9ad", "historyId": "34f6ee4c67ad941177730e21732b3a79", "testCaseId": "c2811b1721d8231bbb0fb40778ae099c", "fullName": "Tests.test_Login.TestLogin#test_2", "labels": [{"name": "severity", "value": "normal"}, {"name": "feature", "value": "Login"}, {"name": "tag", "value": "flaky(reruns=3, reruns_delay=0.5, rerun_except='assert')"}, {"name": "parentSuite", "value": "Tests"}, {"name": "suite", "value": "test_Login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "fv-az570-505"}, {"name": "thread", "value": "1831-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Tests.test_Login"}]}